lib/flame_on/component/capture_schema.ex

```diff
@@ -0,0 +1,72 @@
+defmodule FlameOn.Component.CaptureSchema do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Ecto.Changeset
+
+  schema "capture" do
+    field :module, :string
+    field :function, :string
+    field :arity, :integer
+    field :timeout, :integer
+  end
+
+  @default_attrs %{module: "cowboy_handler", function: "execute", arity: 2, timeout: 15000}
+
+  def changeset(attrs \\ @default_attrs) do
+    %__MODULE__{}
+    |> cast(attrs, [:module, :function, :arity, :timeout])
+    |> validate_required([:module, :function, :arity, :timeout])
+    |> validate_module()
+    |> validate_function_arity()
+  end
+
+  defp validate_module(%Changeset{valid?: false} = changeset), do: changeset
+
+  defp validate_module(changeset) do
+    module_str = get_field(changeset, :module)
+
+    module =
+      try do
+        String.to_existing_atom(module_str)
+      rescue
+        ArgumentError -> nil
+      end
+
+    if is_nil(module) or
```

Comment:
If `module_str` begins with an uppercase letter, AFAIK it's safe to assume that it is an Elixir module, and hence automatically prepend "Elixir." for the user (if it doesn't exist already), which would be nicer UX.

Reply:
Agreed. I plan to make that swap in an upcoming PR.